<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendWhatsappMessage</class>
 <widget class="QWidget" name="SendWhatsappMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyLabelShine" name="label_background">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>381</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF; 
border-radius: 10px;
/*border: 2px solid #368DCE;*/</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="pb_close">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-radius: 5px;

}
QLabel:hover:!pressed{
	  background-color: rgba(255, 200, 200);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/close.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="my_label" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>42</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>240</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/app_logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pb_aceptar">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius: 3px;
	color:rgb(120, 120, 120);
}
QPushButton:hover:!pressed{
	  background-color: #EEEEEE; 
}</string>
      </property>
      <property name="text">
       <string>ENVIAR</string>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/icons/send.png</normaloff>:/icons/send.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pb_cancelar">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius: 3px;
	color:rgb(120, 120, 120);
}
QPushButton:hover:!pressed{
	  background-color: #EEEEEE; 
}</string>
      </property>
      <property name="text">
       <string>CANCELAR </string>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="pt_message">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{
	border-radius: 8px;
	color:rgb(77, 77, 77);
	background-color: #EEEEEE; 
}
</string>
   </property>
   <property name="plainText">
    <string>http://mraguas.com</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="cb_phone_1">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton{	
	color:rgb(120, 120, 120);
	border-radius: 6px;
}

QRadioButton::indicator::checked{
    width: 25px;
	height: 25px;
	image: url(:/icons/checked.png);
}
QRadioButton::indicator::unchecked{
    width: 25px;
	height: 25px;
	image: url(:/icons/unchecked.png);
}

QRadioButton:hover:!pressed{
	  background-color: #EEEEEE; 
}</string>
      </property>
      <property name="text">
       <string>Teléfono 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="cb_phone_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton{	
	color:rgb(120, 120, 120);
	border-radius: 6px;
}

QRadioButton::indicator::checked{
    width: 25px;
	height: 25px;
	image: url(:/icons/checked.png);
}
QRadioButton::indicator::unchecked{
    width: 25px;
	height: 25px;
	image: url(:/icons/unchecked.png);
}

QRadioButton:hover:!pressed{
	  background-color: #EEEEEE; 
}</string>
      </property>
      <property name="text">
       <string>Teléfono 2</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>cb_phone_2</zorder>
   <zorder>cb_phone_1</zorder>
  </widget>
  <widget class="MyLabelSpinner" name="l_spinner_country">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: #EFEFEF; 
	border-radius: 6px;
	color:rgb(50, 50, 50);
}
QLabel:hover:!pressed{
	  background-color: #DDDDDD; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelSpinner</class>
   <extends>QLabel</extends>
   <header location="global">mylabelspinner.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelShine</class>
   <extends>QLabel</extends>
   <header location="global">mylabelshine.h</header>
  </customwidget>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
